src/window.rs

```diff
@@ -120,6 +111,15 @@ impl<'a> From<&'a EdwardsPoint> for LookupTable<AffineNielsPoint> {
     }
 }
 
+impl<T> Zeroize for LookupTable<T>
+where
+    T: Copy + Default + Zeroize
```

Comment:
Could this just be
```
impl<T: Zeroize> Zeroize for LookupTable<T>
```
so that only the `Zeroize` impl required the table entries to be zeroized?  Or are the `Copy + Default` bounds used somewhere in the array impl?

If only this impl had a `Zeroize` bound, then it seems like `LookupTable<T>` would not need `T: Zeroize` and we could skip adding new bounds to some of the where clauses above.

Reply:
Yep, you're right, PR updated

Reply:
Great, happy to merge on green CI